app/javascript/components/journey/overview/learning-section/TrackSummary.tsx

```diff
@@ -32,16 +34,64 @@ export const TrackSummary = ({
             </p>
           </div>
         </div>
-        <div className="velocity-area">
-          <GraphicalIcon icon="velocity" />
-          <div className="journey-h3">{track.velocity}</div>
-          <h4>Progression Velocity</h4>
-          <div className="note">Avg. on Exercism = {avgVelocity}</div>
-          <div className="info">
-            This is a measure of how quickly you’ve progressed through the track
-            in the last 6 months
-          </div>
+        <div className="date-area">
+          <GraphicalIcon icon="entry" />
+          <h4 className="journey-h3">
+            {timeFormat(track.startedAt, 'DD MMM YYYY')}
+          </h4>
+          <p className="text-h6">When you joined the {track.title} Track</p>
+          <p>
+            You started working through the {track.title} Track{' '}
```

Comment:
@iHiD I changed the copy a bit on this in order for us to use our `fromNow` helper.